src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  padding: 50px 40px;
+  gap: 46px 48px;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
+
```

Comment:
Move catalog styles to the separate file

Reply:
Not fixed